projects/storefrontapp/src/environments/models/feature.model.ts

```diff
@@ -1,5 +1,5 @@
 import { ModuleWithProviders, Type } from '@angular/core';
 
 export interface FeatureEnvironment {
-  imports: Type<any>[] | ModuleWithProviders<any>[];
+  imports: (any[] | Type<any> | ModuleWithProviders<{}>)[];
```

Comment:
Why do we need this change?

Reply:
Because otherwise I need to have the 
{ ngModule: TextfieldConfiguratorModule } notation that you don't like :-)

So I copied the signature from the module imports